documentation/book/proc-bridge-retrieving-latest-messages-from-consumer.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-bridge-quickstart.adoc
+
+[id='proc-bridge-retrieving-latest-messages-from-consumer-{context}']
+= Retrieving the latest messages from a Kafka Bridge consumer
+
+Next, retrieve the latest messages from a Kafka Bridge consumer by requesting data from the link:https://strimzi.io/docs/bridge/latest/#_poll[records^] endpoint. For production use, HTTP clients can retrieve messages repeatedly or as a one-off operation. 
```

Comment:
"HTTP clients can retrieve messages repeatedly or as a one-off operation." not sure what you mean here :-(

Reply:
Neither do I! I will update to reflect the recent Strimzi training lab:

"In production use, HTTP clients can call this endpoint repeatedly (in a loop)."